web/src/stream_data.ts

```diff
@@ -864,6 +864,7 @@ export function get_options_for_dropdown_widget(): {
     stream: StreamSubscription;
 }[] {
     return subscribed_subs()
```

Comment:
We could probably filter out archived streams in `subscribed_subs` itself or pass a parameter to it for deciding whether we should include archived streams or not, but that would need a careful audit and can be done next.

Reply:
Yeah probably filtering there would be better.